src/main/java/com/hotels/beeju/BeejuJUnitRule.java

```diff
@@ -16,13 +16,13 @@
 package com.hotels.beeju;
 
 import java.io.File;
+import java.io.IOException;
 import java.util.Map;
 
 import org.apache.hadoop.hive.conf.HiveConf;
 import org.apache.hadoop.hive.metastore.HiveMetaStoreClient;
 import org.apache.thrift.TException;
 import org.junit.rules.ExternalResource;
-import org.junit.rules.TemporaryFolder;
 
 import com.google.common.annotations.VisibleForTesting;
```

Comment:
Not sure how this import stayed here because the save actions should've removed it, but it should be removed :)